packages/flutterfire_cli/lib/src/commands/config.dart

```diff
@@ -170,8 +189,46 @@ class ConfigCommand extends FlutterFireCommand {
     return argResults!['apply-gradle-plugins'] as bool;
   }
 
-  bool get generateAppIdJson {
-    return argResults!['app-id-json'] as bool;
+  bool get generateDebugSymbolScript {
+    return argResults!['debug-symbols-script'] as bool;
+  }
+
+  String? get macosServiceFilePath {
+    if (updatedMACOSServiceFilePath != null) {
+      return updatedMACOSServiceFilePath;
+    }
+    return argResults!['macos-out'] as String?;
+  }
+
+  // This allows us to update to the required "GoogleService-Info.plist" file name for macOS target or scheme writes.
+  String? updatedMACOSServiceFilePath;
+
+  String? get fullMACOSServicePath {
```

Comment:
Should we have macOS in lower case? Uppercase makes me think of the MAC address of a computer
```suggestion
  String? get fullmacOSServicePath {
```